### PR TITLE
feat(Task-1): As a developer, I want a solid start point for the Weather Project

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+module.exports = {
+    entry: './src/app/app.module.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, './dist')
+    },
+    devServer: {
```
Comment (diegopf) [inline config/webpack.config.js]: FYI as a good practice you should split your specific environment configuration into different files.

**Review by diegopf — APPROVED** [review]

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Hello AngularJS!</title>
+</head>
+
+<body body ng-app="app" ng-strict-di ng-cloak>
```
Comment (diegopf) [inline src/index.html]: I forgot, remove the body attribute

**Review by diegopf — CHANGES_REQUESTED** [review]
Pleas, teake a look at https://github.com/jLagosCarrera/TuTiempo/pull/2#discussion_r279792592

File: src/index.html (hunk 2 above)
Comment (eloydotco) [reply src/index.html]: Same as Diego, remove body attribute

**Review by eloydotco — APPROVED** [review]

File: config/webpack.config.js (hunk 1 above)
Comment (jLagosCarrera) [reply config/webpack.config.js]: Already split both files for production and development

**Comment by jLagosCarrera** [comment]
Removed body attribute on index body and added HtmlWebpackPlugin to serve index.html

**Review by diegopf — APPROVED** [review]
I'd suggest you to include the dist folder into the gitignore file

**Comment by jLagosCarrera** [comment]
Requested changes were made, added dist folder to gitignore and deleted it

**Review by diegopf — APPROVED** [review]

**Review by eloydotco — APPROVED** [review]

**Pull request merged**